packages/governance/test/unitTests/test-committee.js

```diff
@@ -81,7 +82,7 @@ test('committee-open question:one', async t => {
     maxChoices: 1,
     maxWinners: 1,
     closingRule: {
-      timer: buildManualTimer(t.log),
+      timer: harden(buildManualTimer(t.log)),
```

Comment:
Shouldn't the `harden` be above on line 77 *instead* of here?

```suggestion
  const questionSpec = coerceQuestionSpec(harden({
      // ...
      timer: buildManualTimer(t.log),
```

Also, does `buildManualTimer` return an unhardened result? That would be surprising, and likely a bug.

I just looked, and `buildManualTimer` does return the result of a `Far` call expression, which is already hardened, so this `harden` call should disappear anyway.

Reply:
Oh, yeah, it looks like I applied the harden as low as possible, and yes it should go higher. I'll put that in the followup PR.

Reply:
huh, actually it doesn't need to be in either place, nor does the one on `issue: harden({ text: 'why' })`. I'm not sure why, maybe `coerceQuestionSpec` does it?

I'll remove it from `timer` but leave it on `issue`, to match the other tests in that file.